.github/workflows/CI.yaml

```
@@ -54,25 +54,16 @@ jobs:
       - name: Refine test environment
         shell: bash -l {0}
         run: |
-          # Install openmm-forcefields via pip (until available via omnia)
-          conda install --yes tinydb "openforcefield>=0.6.0" "openforcefields>=1.0.0"
-          pip install https://github.com/openmm/openmm-forcefields/archive/master.tar.gz
           case ${{ matrix.cfg.openmm }} in
             latest)
               echo "Using latest release OpenMM."
-              conda install --quiet -c omnia openmm;;
-            rc)
-              echo "Using OpenMM rc"
-              conda install --quiet -c omnia/label/rc openmm;;
-            beta)
-              echo "Using OpenMM beta"
-              conda install --quiet -c omnia/label/beta openmm;;
+              conda install --quiet -c conda-forge openmm;;
+           #rc) # We can uncomment this case when we have this label on conda-forge
+             #echo "Using OpenMM rc" 
+             #conda install --quiet -c conda-forge/label/openmm-rc openmm;;
```

As per https://github.com/conda-forge/cfep/blob/master/cfep-05.md#specification : 

```suggestion
             #conda install --quiet -c conda-forge/label/openmm_rc openmm;;
```